runtime/runtime_darwin.go

```diff
@@ -74,6 +74,7 @@ func WaitForLoginCompletion(timeout time.Duration) error {
 	for time.Now().Before(deadline) {
 		username, err := InteractiveUsername()
 		if err != nil {
+			log.Printf("WARNING: Error checking for interactive user: %v", err)
```

Comment:
Weirdly in CI I saw that sometimes `InteractiveUsername()` was called twice, and I've no idea why it failed the first time, so this should help diagnose.